src/routes/boot.ts

```diff
@@ -334,6 +334,31 @@ const getExperimentation = async (
   };
 };
 
+const getUser = (con: DataSource, userId: string): Promise<User> =>
+  con.getRepository(User).findOne({
+    where: { id: userId },
+    select: [
+      'id',
+      'username',
+      'name',
+      'email',
+      'image',
+      'company',
+      'title',
+      'infoConfirmed',
+      'notificationEmail',
+      'acceptedMarketing',
```

Comment:
Do we need these in the fe from boot?

Reply:
So, I didn't want to take risks in this PR, but I definitely know we don't need the new fields I added, which can be quite big. Hence, I manually selected what we had before. I think we should optimize it and see what's needed, but that's out of this scope. The previous implementation just selected everything